kvm-bindings/src/lib.rs

```diff
@@ -18,6 +18,8 @@ extern crate serde;
 #[cfg(feature = "serde")]
 extern crate zerocopy;
 
+extern crate core;
```

Comment:
why do we need this? :o

Reply:
whew. I'll submit a PR to get us onto edition 2024 tmrw, no reason for us to be on 2015 except that no one ever bothered to update it I guess (we dont really have a MSRV policy around here either)

Reply:
For the record, I've created #329 so that we don't forget it